Source/WTF/wtf/StdLibExtras.h

```diff
@@ -846,23 +846,19 @@ inline void* operator new(size_t, NotNullTag, void* location)
     return location;
 }
 
-namespace std {
+namespace WTF {
 
 template<WTF::CheckMoveParameterTag, typename T>
```

Comment:
I don't think we need `CheckMoveParameter` if we aren't trying to define `move` in the `std` namespace.